runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/state/InMemoryBagUserStateFactory.java

```diff
@@ -41,9 +41,12 @@
 public class InMemoryBagUserStateFactory<K, V, W extends BoundedWindow>
```

Comment:
You should update the comment related to error handling to make sure that users discard the factory every time they want a new cache token.

Reply:
Sure, good idea to clarify that.